src/cli/command-package.ts

```diff
@@ -29,8 +29,18 @@ export default async function run(options: CliOptions) {
 
   // Ok, we're updating a package in a directory not under our control.
   // We need to be sure it's safe.
-  const {stdout, stderr} = await exec(inDir, 'git', ['status', '-s']);
-  if (!options.force && (stdout || stderr)) {
+  let stdout, stderr;
+  try {
+      ({stdout, stderr} = await exec(inDir, 'git', ['status', '-s']));
+  } catch (e) {
+      ({stdout, stderr} = e);
+  }
+  const isRepo = !((stderr || '').indexOf('Not a git repository') !== -1);
```

Comment:
Looks like double-negation here, can't this be:

```ts
const isRepo = (stderr || '').indexOf('Not a git repository') === -1;
```

In addition, we prefer explicit checks to boolean conversions, so:

```ts
const isRepo = stderr === undefined || stderr.indexOf('Not a git repository') === -1;
```